core/domain/state_domain.py

```diff
@@ -564,6 +620,14 @@ def get_all_html_content_strings(self):
         if self.solution:
             solution_html = self.solution.explanation.html
             html_list = html_list + [solution_html]
+            if self.solution.correct_answer:
+                interaction = (
+                    interaction_registry.Registry.get_interaction_by_id(
+                        self.id).to_dict())
+                if (interaction['answer_type'] ==
+                        feconf.ANSWER_TYPE_LIST_OF_SETS_OF_HTML):
+                    for value in self.solution.correct_answer:
+                        html_list = html_list + [value[0]]
```

Comment:
Why value[0]? I was expecting value to be a string

Reply:
Yup I'm wondering about this as well. In general stuff like "[0]" is a bit of a red flag.

Reply:
value is not a `string` its actually a set of HTML as the name suggests(ListOfSetOfHtml).
I used value[0] because this set's length for dragAndDrop interaction is always one.
I've removed the hardcoding and made it general.